arrays-and-scope.js

```diff
@@ -6,17 +6,30 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // verwacht antwoord: 6
 // --------------------------
 
+found=0;
```

Comment:
Ik zou hier nog even `let` voor zetten. In dit geval maakt het niet zoveel uit, want de variable zit toch in de global scope.